ee/server/service/mdm.go

```diff
@@ -933,6 +943,7 @@ func (svc *Service) getOrCreatePreassignTeam(ctx context.Context, groups []strin
 				// instead by CopyDefaultMDMAppleBootstrapPackage below
 				// BootstrapPackage:            ac.MDM.MacOSSetup.BootstrapPackage,
 				EnableEndUserAuthentication: ac.MDM.MacOSSetup.EnableEndUserAuthentication,
+				// TODO(mna): should we copy the EnableReleaseDeviceManually setting from the global config?
```

Comment:
@noahtalerman @roperzh This is the settings used when creating a new team from a Puppet run (the preassignment step), it already copies the MacOS Setup Assistant and End-user authentication settings from the no-team (global) config. Should it also copy the new `EnableReleaseDeviceManually` global setting?

Note that `ModifyTeam` (called next) only supports the end-user authentication update, not the `macos_setup_assistant`, so I don't think this copies that path information unless I'm missing something.

Reply:
For the usage the customer is doing I think it should mirror it as well 👍 . They want to:

1. Define the ABM settings in the "default ABM team"
2. Make sure that hosts still have those settings, even after the Puppet module reassigns the host to a different team.

Reply:
Gotcha, thanks! I think this will require switching from `ModifyTeam` to `ApplyTeamSpecs` so that the whole range of settings can be updated.

Reply:
damn, sorry about that! I remember we do special handling for the bootstrap package, maybe for this same reason.

Reply:
Quite possibly. Yeah this is very error-prone, the fact that `ModifyTeam` only handles few if any configs (it was none at first I think, but disk encryption was moved and was added there).

Reply:
So as mentioned in standup, if you don't mind I'll address that change in a future PR, so we can land this in the feature branch and Gabe can integrate the UI with the API changes.

Reply:
absolutely! makes sense!